proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -226,39 +204,43 @@ The above code is sugar for:
 
 ```swift
 func insert(text: String) {
-  @Logged var text = text
+  let _text: Logged<String> = Logged(wrappedValue: text)
+
+  var text: String { _text.wrappedValue }
 }
 ```
 
-which the compiler further desugars to generate:
+Note that the backing storage is a let-constant, and the local `text` property does not have a setter.
 
-```swift
-func insert(text: String) {
-  var _text: Logged<String> = Logged(wrappedValue: text)
+> **Rationale**: It would be confusing if it appears that a user can mutate a parameter, because those changes will not be reflected in the caller once the function returns.
```

Comment:
Should we mention [SE-0003](https://github.com/apple/swift-evolution/blob/main/proposals/0003-remove-var-parameters.md)?

---
(I've actually made some more changes to match more closely SE-0003's motivation)

**Rationale**: The ability to mutate a wrapped parameter would likely confuse users into thinking that the mutations they make are observable by the caller; that's not the case. In fact, there was a similar feature in this language which was removed in [SE-0003](https://github.com/apple/swift-evolution/blob/main/proposals/0003-remove-var-parameters.md).